.github/buildomat/jobs/deploy.sh

```diff
@@ -5,7 +5,8 @@
 #: target = "lab-opte-0.21"
 #: output_rules = [
 #:	"%/var/svc/log/system-illumos-sled-agent:default.log",
-#:	"%/zone/oxz_nexus/root/var/svc/log/system-illumos-nexus:default.log",
+#:	"%/zone/oxz_*/root/var/svc/log/system-illumos-*.log",
+#:	"!/zone/oxz_propolis-server_*/root/var/svc/log/*.log",
```

Comment:
What's the difference between a line starting with "%" and "!" here?

Reply:
`%` causes matching files to be uploaded on a best-effort basis, meaning that if they change (or are removed, after oxidecomputer/buildomat#16), the job won't fail.

`!` negates any other pattern, meaning that if any file matches a `!` pattern, it will not be uploaded even if it also matches a regular pattern (no prefix or `%`).  We have this here for propolis basically to work around buildomat#16.

Reply:
(deeper reference: https://github.com/oxidecomputer/buildomat#configuration, "output rules")